sql-mode.md

```diff
@@ -27,7 +27,7 @@ Ensure that you have `SUPER` privilege when setting SQL mode at `GLOBAL` level,
 | `PIPES_AS_CONCAT` | Treats "\|\|" as a string concatenation operator (`+`) (the same as `CONCAT()`), not as an `OR` (full support) |
 | `ANSI_QUOTES` | Treats `"` as an identifier. If `ANSI_QUOTES` is enabled, only single quotes are treated as string literals, and double quotes are treated as identifiers. Therefore, double quotes cannot be used to quote strings. (full support)|
 | `IGNORE_SPACE` | If this mode is enabled, the system ignores space. For example: "user" and "user " are the same. (full support)|
-| `ONLY_FULL_GROUP_BY` | If a non-aggregated column that is referred to in `SELECT`, `HAVING`, or `ORDER BY` is absent in `GROUP BY`, this SQL statement is invalid, because it is abnormal for a column to be absent in `GROUP BY` but displayed by query. (full support) |
+| `ONLY_FULL_GROUP_BY` | If columns not processed by aggregate functions or not included in `GROUP BY` appear in `SELECT`, `HAVING`, or `ORDER BY`, the SQL query is considered invalid because it is abnormal for such columns to be displayed by query (full support). |
```

Comment:
Maybe something like this?
```suggestion
| `ONLY_FULL_GROUP_BY` | If columns neither use an aggregate functions nor are included in `GROUP BY` appear in `SELECT`, `HAVING`, or `ORDER BY`, the SQL query is considered invalid because it is likely an incorrect result. |
```

The problem is that a column isn't in `GROUP BY` nor uses a aggregate query then there can be multiple values and if the database picks one then the result is probably incorrect.